web/pdf_page_view.js

```diff
@@ -183,6 +184,8 @@ class PDFPageView {
     });
     this.div = div;
 
+    this._accessibilityManager = new AccessibilityManager(div);
+
```

Comment:
We probably don't want to initialize this unconditionally, since the textLayer may not actually be used (depending on options). It ought to suffice, as far as I can tell, to initialize it a the top this block: https://github.com/mozilla/pdf.js/blob/4f6cd05a53e6c069bb4ebee469fda6437a0971af/web/pdf_page_view.js#L699